.gitignore

```diff
@@ -48,3 +48,5 @@ Makefile.local
 .ycm_extra_conf.pyc
 __pycache__
```

Comment:
These too are python compiled files. Can you move it below the comment (and remove the one `pyc` file above)?